truss/templates/vllm/vllm.Dockerfile.jinja

```diff
@@ -2,19 +2,33 @@ FROM baseten/vllm:latest
 
 EXPOSE 8080-9000
 
+{% if hf_access_token %}
+ENV HUGGING_FACE_HUB_TOKEN {{hf_access_token}}
+{% endif %}
+
+COPY ./cache_requirements.txt cache_requirements.txt
+RUN pip install -r cache_requirements.txt --no-cache-dir && rm -rf /root/.cache/pip
+
+COPY ./data /app/data
+
+{%- if hf_cache != None %}
+COPY ./cache_warmer.py /cache_warmer.py
+RUN chmod +x /cache_warmer.py
```

Comment:
this line is not necessary since you're doing `python3 /cache_warmer.py` below. `+x` is only needs if you wanna execute the file directly